layouts/shortcodes/signatures.html

```diff
@@ -1,19 +1,12 @@
+{{ $names := .Site.Data.names.signatures.list }}
 <div>
-    {{ with $.Site.Data.names.signatures }}
-    {{ $list := .list }}
-    {{ $len := (len $list) }}
-
-    <h4>({{ $len }})</h4>
-        {{ range $index, $element := $list }}
-            {{ if gt $index 0 }}<bold> &bull; </bold>{{ end }}
-                {{ $fullName := split .name  " " }}
-                {{ if .contact }}
-                    <a href="{{ .contact }}" alt="{{ .name }}" target="_blank">
-                        {{ range $fullName }}{{ . }}&nbsp;{{ end }}
```

Comment:
Can you re-add the non-breaking space to prevent names from wrapping between lines? Otherwise, GTG.

Reply:
Fixed in https://github.com/Minimum-CD/cd-manifesto/pull/308/commits/6d9e95b78e9c395652e63b879b5f926ee809d265 :+1: